MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/javaservices/MatrixApiActivity.java

```diff
@@ -205,24 +206,24 @@ private void initPositionListFromGeoJsonFile() {
     featureCollection = FeatureCollection.fromJson(loadGeoJsonFromAsset("boston_charge_stations.geojson"));
 
     // Initialize List<Position> for eventual use in the Matrix API call
-    positionList = new ArrayList<>();
+    pointList = new ArrayList<>();
 
     // Get the position of each GeoJSON feature and build the list of Position
     // objects for eventual use in the Matrix API call
-    for (Feature singleLocation : featureCollection.getFeatures()) {
-      Position singleLocationPosition = (Position) singleLocation.getGeometry().getCoordinates();
-      positionList.add(singleLocationPosition);
+    for (Feature singleLocation : featureCollection.features()) {
+      Point singleLocationPoint = (Point) singleLocation.geometry();
+      pointList.add(singleLocationPoint);
```

Comment:
You should add the item directly to the list rather than declaring a local variable and then passing it in.

Reply:
Fixed.